src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/AIReviewerApiApplication.java

```diff
@@ -4,10 +4,10 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class EfilingReviewerApiApplication {
+public class AIReviewerApiApplication {
```

Comment:
Same as above.

Reply:
yeah, that's good. Updated name to AiReviewerApiApplication